src/main/webapp/resources/js/pages/analysis/shared-preview.js

```diff
@@ -0,0 +1,18 @@
+import { convertFileSize } from "../../utilities/file.utilities";
+
+/**
+ * Status text for showing how many bytes of a file have been loaded.
+ * @param {number} byte Number of bytes of file currently fetched from server
+ * @param {number} fileSizeBytes File size in bytes
+ * @returns {string}
+ */
+export function statusText(byte, fileSizeBytes) {
```

Comment:
This function is incorrectly named I think.  It is not returning `statusText` nor what the description says.  If I am not mistaken, it is return a percentage - of the file being downloaded.  Can you rename it, and update the comment.

These both seem like good utility function to have available everywhere.  Can you put the in the `file.utilities` file directly.  Then rename `file.utilities` to `file-utilities` since it does not match the naming structure of all the other utility files.

Reply:
Sure. Yeah it's what was in the json-preview and plaintext-preview files previously.

Reply:
Updated in 17c23fd74ab4bdcfe6c55f80cbe3c82226c59dab